util/archive/archive_test.go

```diff
@@ -56,7 +57,7 @@ func TestTarFile(t *testing.T) {
 	log.Infof("Taring to %s", f.Name())
 	w := bufio.NewWriter(f)
 
-	err = TarGzToWriter(data.Name(), w)
+	err = TarGzToWriter(data.Name(), gzip.DefaultCompression, w)
```

Comment:
For a test I think it would be enough to duplicate this with a different compression